lib/main.coffee

```diff
@@ -77,9 +88,10 @@ module.exports =
             '--output-format=text'
           ]
           if @rcFile
-            args.push "--rcfile=#{@rcFile}"
+            rcFile = @rcFile.replace(/%p/g, projDir).replace(/%f/g, path.dirname(file))
```

Comment:
Earlier you used a substring, here you are using a regex, as far as i can tell a substring should work just fine here as well.